scripts/bootstrap.sh

```diff
@@ -140,3 +140,7 @@ if [ -z ${AIRY_VERSION+x} ]; then
 fi
 
 AIRY_VERSION=${AIRY_VERSION} vagrant up
+
+mkdir -p ~/.airy
+cd $infra_path
+vagrant ssh -c "cat /etc/rancher/k3s/k3s.yaml" 2>/dev/null | sed "s/127.0.0.1/192.168.50.5/g" > ~/.airy/kube.conf
```

Comment:
We can also do this with a provisionner, this is more lightweight I think, because otherwise we need to install another plugin, to be able to run commands on the host (the shell provisionner runs commands only on the guest vm).